src/catalogue/items/catalogueItemsTable.component.test.tsx

```diff
@@ -200,14 +200,6 @@ describe('Catalogue Items Table', () => {
 
     await ensureColumnsVisible(['Description']);
 
-    await waitFor(() => {
-      expect(
-        screen.getByLabelText(
-          'Catalogue item description: Precision energy meters for accurate measurements. 26'
-        )
-      ).toBeInTheDocument();
-    });
```

Comment:
Was this not needed to prevent test flakiness by ensuring it finishes loading?

Reply:
The test seems to work fine without it, can add it back in though to prevent flakiness